receiver/mysqlreceiver/metadata.yaml

```diff
@@ -283,6 +287,16 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.opened:
```

Comment:
Since we have `tmp_resources`, should this be `opened_resources`?

Or, should we take the opportunity to make a `mysql.resource` namespace and rename both metrics onto that?

Reply:
> Or, should we take the opportunity to make a mysql.resource namespace and rename both metrics onto that?

It doesn't seem to be the same kind of resources